src/containers/ListingPage/listingSagas.js

```diff
@@ -21,6 +21,11 @@ export function* fetchListing() {
   if (!listings.results) {
     yield put(actions.setListing([]));
   } else {
+    const numberOfPages = Math.ceil(listings.totalCount / listings.pageSize);
+    for (let i = 2; i <= numberOfPages; i += 1) {
```

Comment:
Selv om jeg endrer på `listingApi` så vil ikke dette ha noen innvirkning på `listingSagas`.
Linje 19 vil alltid hente inn side 1, og får dermed også informasjon om hvor mange elementer (totalt, og per side) som finnes, mens for-løkken på linje 25-28 henter inn alt fra side 2 og utover hvis nødvendig.